app/common/state/tabState.js

```diff
@@ -610,7 +610,7 @@ const tabState = {
     state = tabState.removeTabField(state, 'messageBoxDetail')
     state = tabState.removeTabField(state, 'frame')
     state = state.delete('tabsInternal')
-    return state.delete('tabs')
+    return state.set('tabs', Immutable.List())
```

Comment:
because we used to set default app state in 2 places but now just one place.  Via Object.assign after a clean, but I'd of had to have 2 different methods for that, one for immutable and one for not.